src/main/java/seedu/budgetbaby/ui/MainWindow.java

```diff
@@ -225,6 +225,7 @@ private CommandResult executeCommand(String commandText) throws CommandException
             resultDisplay.setFeedbackToUser(commandResult.getFeedbackToUser());
 
             if (commandResult.isRefreshUi()) {
+                System.out.println(logic.getFilteredMonthList());
```

Comment:
@eksinyue I think you can use the `logger` instance to print out your debug messages.

Reply:
Sorry for pointing it out. I forgot to remove it before pushing.